app/components/show-header/template.hbs

```diff
@@ -16,13 +16,13 @@
     links=links
     activeTabIndex=activeTabIndex}}
 
-  <div class="donate-button-link">
-    <a href="https://pledge3.wqxr.org/donate/main/onestep/?ref=button-donate-header" target="_blank">
-      <button class="donate-button">
-        Donate
-      </button>
-    </a>
-  </div>
-
 </div>
-{{/sticky-element}}
\ No newline at end of file
+{{/sticky-element}}
+
+<div class="donate-button-link">
+  <a href="https://pledge3.wqxr.org/donate/main/onestep/?ref=button-donate-header" target="_blank">
+    <button class="donate-button">
+      Donate
+    </button>
+  </a>
+</div>
```

Comment:
did you intentionally move the donate button outside of the sticky element instead of removing it?

Reply:
The way the donate button was set up was that the same donate div was being used for when it was rendered on the static vs persistent sticky. So by taking it out of the ember-sticky component, it will no longer render when there is a sticky header, which is what we want for now.